index.bs

```diff
@@ -585,22 +585,25 @@ dictionary MLOperandDescriptor {
 </script>
 
 ## MLOperand ## {#api-mloperand}
+
+The {{MLOperand}} interface represents data that flows within the computational graph. See [[#programming-model]].
+
 <script type=idl>
 [SecureContext, Exposed=(Window, DedicatedWorker)]
 interface MLOperand {};
 </script>
 
 ## MLOperator ## {#api-mloperator}
 
-The {{MLOperator}} interface defines a type of operation such as the various types of activation function used to create other operations such as [[#api-mlgraphbuilder-conv2d]] or [[#api-mlgraphbuilder-batchnorm]].
+Objects implementing the {{MLOperator}} interface represent activation function types: {{clamp()}}, {{elu()}}, {{hardSigmoid()}}, {{hardSwish()}}, {{leakyRelu()}}, {{linear()}}, {{relu()}}, {{sigmoid()}}, {{softplus()}}, {{softsign()}}, {{tanh()}}.
```

Comment:
Listing the available activation functions would help readers who jump from the operations that support the fused activations, such as `conv2d` or `batchNormalization`. However, I am not sure whether we should keep mentioning these operations. It might be useful for readers who first read the `MLOperator`, so they can jump to `conv2d` or `batchNormalization` to get an idea of how these activation functions are used.

Reply:
Thanks for the comment. My idea was to enable navigation from ops supporting fused activations to activation functions. I thought we'd use MLOperator as an intermediary. I'm hearing navigating from MLOperator to supporting ops would also be helpful, and it indeed makes sense to enable cross-linking both ways.

Should we add a complete list of such ops here or is the "such as § 7.7.4 conv2d or § 7.7.1 batchNormalization" good enough?

I pushed an update where the list of activation function becomes an informative note, also noted what type of operations support activations. PTAL.

We could expand that note with a few examples or a complete list of ops that support fused activations.

WDYT?